shell/yq.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Wrapper around `yq` that allows use of python-yq or gojq.
+# You can force using gojq by setting the environment variable YQ_USE_GOJQ=true.
+#
+
+set -euo pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+GOJQ_VERSION="${GOJQ_VERSION:-$(grep ^gojq: "$DIR"/../versions.yaml | awk '{print $2}')}"
+
+use_gojq=false
+
+if [[ -n $YQ_USE_GOJQ ]]; then
+  use_gojq=true
+elif command -v yq >/dev/null 2>&1; then
```

Comment:
```suggestion
 # Make sure it's the correct yq. The Go yq is not compatible with jq syntax.
elif ! (command -v yq >/dev/null 2>&1 && [[ "$(yq --version)" =~ ^yq.3.* ]]);  then
  use_gojq=true
```
Suggest: could make this conditional check simplified. (Github does not allow to choose multiple lines to suggest somehow. )

Reply:
I'm not sure if that conditional is actually simpler, since you're negating a predicate which is already pretty complicated - it's a mess of symbols. I'd rather be more verbose when possible with bash for this reason.

Reply:
That's fair.